doc/spec/estimation/forest.rst

```diff
@@ -124,32 +131,35 @@ Similarly, we can call :py:class:`~econml.ortho_forest.DiscreteTreatmentOrthoFor
     >>> T = np.array([0, 1]*60)
     >>> W = np.array([0, 1, 1, 0]*30).reshape(-1, 1)
     >>> Y = (.2 * W[:, 0] + 1) * T + .5
-    >>> est = DiscreteTreatmentOrthoForest(n_trees=1, max_splits=1, subsample_ratio=1,
+    >>> est = DiscreteTreatmentOrthoForest(n_trees=1, max_depth=1, subsample_ratio=1,
     ...                                    propensity_model=sklearn.linear_model.LogisticRegression(),
     ...                                    model_Y=sklearn.linear_model.LinearRegression())
     >>> est.fit(Y, T, W, W)
+    <econml.ortho_forest.DiscreteTreatmentOrthoForest object at 0x...>
     >>> print(est.effect(W[:2]))
-    [[1. ]
-     [1.2]]
+    [1.  1.2]
 
 Let's now look at a more involved example with a high-dimensional set of confounders :math:`W`
 and with more realistic noisy data. In this case we can just use the default parameters
 of the class, which specify the use of the :py:class:`~sklearn.linear_model.LassoCV` for 
 both the treatment and the outcome regressions, in the case of continuous treatments.
 
-    >>> X = np.random.uniform(-1, 1, size=(4000, 1))
-    >>> W = np.random.normal(size=(4000, 50))
-    >>> support = np.random.choice(50, 4, replace=False)
-    >>> T = np.dot(W[:, support], np.random.normal(size=4)) + np.random.normal(size=4000)
-    >>> Y = np.exp(2*X[:, 0]) * T + np.dot(W[:, support], np.random.normal(size=4)) + .5
-    >>> est = ContinuousTreatmentOrthoForest()
-    >>> est.fit(Y, T, X, W)
-    >>> X_test = np.linspace(-1, 1, 30).reshape(-1, 1)
-    >>> treatment_effects = est.effect(X_test)
-    >>> plt.plot(X_test, y, label='ORF estimate')
-    >>> plt.plot(X_test[:, 0], np.exp(2*X_test[:, 0]), 'b--', label='True effect')
-    >>> plt.legend()
-    >>> plt.show()
+    .. doctest::
+        :options: +SKIP
+
+        >>> X = np.random.uniform(-1, 1, size=(4000, 1))
+        >>> W = np.random.normal(size=(4000, 50))
+        >>> support = np.random.choice(50, 4, replace=False)
+        >>> T = np.dot(W[:, support], np.random.normal(size=4)) + np.random.normal(size=4000)
+        >>> Y = np.exp(2*X[:, 0]) * T + np.dot(W[:, support], np.random.normal(size=4)) + .5
+        >>> est = ContinuousTreatmentOrthoForest()
+        >>> est.fit(Y, T, X, W)
+        >>> X_test = np.linspace(-1, 1, 30).reshape(-1, 1)
+        >>> treatment_effects = est.effect(X_test)
+        >>> plt.plot(X_test, y, label='ORF estimate')
```

Comment:
What happens to plots in doctests? Can we maybe move the plot out of the doctest and just say in words. If we plot X_test vs the predicted effect, here is the figure we get.

Reply:
Doctests test that each command runs without error and that any printed output matches.  Since `plt.plot` doesn't print any output we will just verify that it runs without error.

However, that doesn't make it bad to include if it's something that we want users to try out, although we can't automatically test that the output visually matches.

Reply:
makes sense. We should still run it (After the mods proposed in the other comment)